<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LiCount</string>
  </property>
  <widget class="QPushButton" name="buttonSelect">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select a folder !</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>38</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelFolder">
      <property name="text">
       <string>Folder: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditPath">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>103</y>
     <width>91</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>Files types:</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>71</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QCheckBox" name="checkC">
       <property name="text">
        <string>*.c</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkCpp">
       <property name="text">
        <string>*.cpp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkH">
       <property name="text">
        <string>*.h</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkUi">
       <property name="text">
        <string>*.ui</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkHtml">
       <property name="text">
        <string>*.html</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkCss">
       <property name="text">
        <string>*.css</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkPhp">
       <property name="text">
        <string>*.php</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>112</y>
     <width>251</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonQuit">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>410</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkSub">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>351</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Add subdirectories</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAbout">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>410</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
